e2e/core-e2e/tests/nx-dotnet.spec.ts

```diff
@@ -503,7 +498,7 @@ function initializeGitRepo(cwd: string) {
   runCommand('git config user.email no-one@some-website.com', {});
   runCommand('git config user.name CI-Bot', {});
   runCommand('git add .', {});
-  runCommand('git commit -m "initial commit"', {});
+  runCommand('git commit -m "initial commit" --no-gpg-sign', {});
```

Comment:
When you use a smart card to sign with GPG, it is painful to touch the key to prevent the tests from being blocked.